src/test/kotlin/org/malachite/estella/people/users/UserIntegration.kt

```diff
@@ -72,40 +72,23 @@ class UserIntegration : BaseIntegration() {
     @Test
     @Order(6)
     fun `should update created User`() {
-        val user = getUsers().find { it.mail == mail }!!
-
-        val response = updateUser(user.id!!)
+        val response = updateUser()
 
         expectThat(response.statusCode).isEqualTo(HttpStatus.OK)
 
-        val updatedUser = getUserById(user.id!!)
+        val updatedUser = getUserById()
         expectThat(updatedUser.firstName == newName)
     }
 
     @Test
     @Order(7)
     fun `should return Bad Request with message that user is unauthenticated`() {
```

Comment:
Ogólnie celem tego testu było sprawdzenie, czy serwis zachowuje się odpowiednio, jeśli chcemy zupdateować usera, który nie istnieje. Spróbuj ten test powtórzyć dla jakiegoś legitnego tokenu, ale usera o id 100000 na przykład

Reply:
Dla użytkownika z id 100000 jak będzie istniał to test przejdzie a jak nie będzie to tak samo wyjdzie że niepoprawny token

Reply:
To ok